index.js

```diff
@@ -47,7 +57,8 @@ module.exports = ({negotiateProtocol}) => {
 
 			const {
 				extensions = ['ts'],
-				rewritePaths: relativeRewritePaths
+				rewritePaths: relativeRewritePaths,
+				compile: relativeCompile = true
```

Comment:
```suggestion
				compile = true
```

What's the meaning of "relative" here?

Reply:
dunno, will fix that